### PR TITLE
Created all six models but without any database interrelationship, review or migrations

### PR DESCRIPTION
**Pull request by collinsezedike**
I suggest we create the school and student as apps instead of models in the same app, as it will enable easy dashboard differentiation.

### CODE REVIEW

**Comment by ebereanuebunwa** [comment]
Yes there will be an app just for accounts(student and school) 
And then an app for the other functionalities.

**Review by zmunetsi — APPROVED** [review]
Hi Collins. You dont need to fork repo. Clone main repo and create branches there.

**Review by Thatslimdev — APPROVED** [review]
Nice work collins

File: Backend/studebt/my_debtors/models.py [hunk 1]
```diff
@@ -1,3 +1,59 @@
...
+    email = models.EmailField(max_length=254)
+    state = models.CharField(max_length=120)
+    lga = models.CharField(max_length=120)
+    contact = models.CharField(max_length=120) 
+    address = models.CharField(max_length=120) 
+    password = models.CharField(max_length=120)
+    date_created = models.DateField(auto_now_add=True)
+
+
+class GlobalStudent(models.Model):
```
Comment (genuineversion) [inline Backend/studebt/my_debtors/models.py:33]: I think this is okay. We can always flesh out the models. Well done Collins

**Review by genuineversion — COMMENTED** [review]
I think this is okay. We can always flesh out the models. Well done Collins

...Based on the agreed models and schema, do we need to still have a separate model for global student id? Since we are not pre-filling (pre-populating) it.

Also, we still have a research deliverable on how to go about the user model and incorporate both the school and student user.

**Pull request merged**